shared/devices/index.js.flow

```diff
@@ -0,0 +1,20 @@
+// @flow
+import {Component} from 'react'
+
+import type {Device} from '../constants/types/flow-types'
+import type {MenuItem} from '../common-adapters/popup-menu.js'
+
+export type Props = {
+  deviceIDs: Array<string>,
+  // $FlowIssue not sure why menuitem conflicts
+  menuItems: Array<MenuItem>,
```

Comment:
Making this `MenuItem | 'Divider' | null` will fix the flow issue. Flow can't generalize `Array<MenuItem>` => `Array<MenuItem | 'Divider' | null>`, which is what the PopupMenu component expects.